frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/Maskinporten.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import { Maskinporten } from './Maskinporten';
+import { renderWithProviders } from '../../../../../../../../test/mocks';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import userEvent from '@testing-library/user-event';
+
+describe('Maskinporten', () => {
+  const consoleLogMock = jest.fn();
+  const originalConsoleLog = console.log;
+  beforeEach(() => {
+    console.log = consoleLogMock;
+  });
+
+  afterEach(() => {
+    console.log = originalConsoleLog;
+  });
+
+  it('should check and verify if the user is logged in', async () => {
+    const getIsLoggedInWithAnsattportenMock = jest
+      .fn()
+      .mockImplementation(() => Promise.resolve(false));
+
+    renderMaskinporten({
+      queries: {
+        getIsLoggedInWithAnsattporten: getIsLoggedInWithAnsattportenMock,
+      },
+    });
+    await waitForLoggedInStatusCheckIsDone();
+    await waitFor(() => expect(getIsLoggedInWithAnsattportenMock).toHaveBeenCalledTimes(1));
+  });
+
+  it('should display information about login and login button, if user is not logged in', async () => {
+    renderMaskinporten({});
```

Comment:
Could we change the params of the function so that we do not need to send in an empty object? 😄 

```suggestion
    renderMaskinporten();
```